dataactvalidator/scripts/read_zips.py

```diff
@@ -418,6 +439,7 @@ def read_zips():
         hot_swap_zip_tables(sess)
         update_state_congr_table_current(sess)
         update_state_congr_table_census(census_file, sess)
+        export_state_congr_table(sess)
```

Comment:
Should this be in an "if" and only done if not running locally? We wouldn't want our locals to accidentally overwrite important things.

Reply:
It's a good point. It'd be super hard nowadays for local users to have AWS access to the buckets but regardless that would prevent them from it being an issue at all.